include/app/config.hpp

```diff
@@ -109,6 +109,19 @@ class Config : public QObject {
         emit page_changed(page, enabled);
     }
 
+    inline QString get_cam_stream_url() {return cam_stream_url;}
```

Comment:
spaces inbetween the parens (same below)

Reply:
Ugh